src/odemis/driver/tescan.py

```diff
@@ -1096,6 +1106,58 @@ def _pollXYZ(self):
         except Exception:
             logging.exception("Unexpected failure during XYZ polling")
 
+    def _checkPosition(self, ref_pos: Dict[str, float], pos: Dict[str, float], rel: bool = False):
+        """
+        Checks if the current stage position is equal to the given position.
```

Comment:
It doesn't just *check*, it also *waits*. Please indicate that in the docstring.